docs/src/confidential-token/deep-dive/encryption.md

```diff
@@ -256,7 +256,7 @@ struct ConfidentialTransferInstructionData {
 ```
 
 The fields `pending_balance_credit_counter` and
-`maximum_pending_balance_credit_counter` are used to limit amounts that are
+`maximum_pending_balance_credit_counter` is used to limit amounts that are
```

Comment:
The subject of this verb is "the fields", so "are" is correct. Please revert this change.